functional_tests.go

```diff
@@ -96,6 +96,19 @@ func successLogger(function string, args map[string]interface{}, startTime time.
 	return log.WithFields(fields)
 }
 
+// As few of the features are not available in Gateway(s) currently, Check if err value is NotImplemented,
+// and log as NA in that case and continue execution. Otherwise log as failure and return
+func logError(function string, args map[string]interface{}, startTime time.Time, alert string, message string, err error) {
+	// If server returns NotImplemented we assume it is gateway mode and hence log it as info and move on to next tests
+	// Special case for ComposeObject API as it is implemented on client side and adds specific error details like `Error in upload-part-copy` in
+	// addition to NotImplemented error returned from server
+	if isErrNotImplemented(err) || err.Error() == "Error in upload-part-copy - A header you provided implies functionality that is not implemented" {
```

Comment:
Unable to get why ignoring based on error string

Reply:
`Compose-Object` is not server side, so it prepends the error string from server with `Error in xyz`, (xyz being the exact server api that failed)

Reply:
On what case we get an error from `ComposeObject()`?

Reply:
You would need to fix by removing err.Error()

Reply:
done